server/src/Types.hs

```diff
@@ -104,6 +107,19 @@ newtype AppliedScript = AppliedScript Ledger.Script
   deriving stock (Show, Generic)
   deriving newtype (Eq, FromJSON, ToJSON)
 
+data FinalizeRequest = FinalizeRequest
+  { tx :: Cbor
+  , datums :: [Cbor]
+  , redeemers :: Cbor
```

Comment:
I think we'll need a different approach for sending the redeemers to the backend. I'll send you a message to discuss